metadata-ingestion/src/datahub/emitter/mcp_builder.py

```diff
@@ -90,6 +90,25 @@ def property_dict(self) -> Dict[str, str]:
     def as_urn(self) -> str:
         return make_container_urn(guid=self.guid())
 
+    def parent_key(self) -> Optional["ContainerKey"]:
+        # Find the immediate base class of self.
+        # This is a bit of a hack, but it works.
+        base_classes = self.__class__.__bases__
+        if len(base_classes) != 1:
```

Comment:
as of today, is there any `ContainerKey` class going through this `if`?

we may add it to the test cases

Reply:
adding it to the tests